docs/disasterRecoverInstructions.md

```diff
@@ -0,0 +1,79 @@
+### Disaster Recovery Instructions
+
+
+
+## Deploy Devhub on OpenShift
+Learning from build.js in pipeline/lib file about its’ deploying process, here’s my approach of manually deploy Devhub. 
+ 
+**prerequisite：**
+Have a nodejs version 10 and s2i-caddy image from Redhat in ‘openshift’ namespace.
+
+Have Openshift Cli knowledge.
+
+Have all required API token.(Github, matomo, eventbrite)
+
+Assuming we start in a empty namespace
+ 
+ 
+In OpenShift: namespace: BC developer Hub(tools)
+
+
+**Step 1, add following secrete objects to this namespace**
+* devhub-eventbrite-token
+* devhub-gh-token
+* devhub-meetup-token
+* matomo-api-key
+
+ we can add by using the corresponding template in the project file: openshift/template:
+
+```cmd
+oc process -n devhub-tools -f gh.secret.yaml -p GITHUB_TOKEN=<token that you get from github> | oc apply -f –
+ 
+oc process -n devhub-tools -f matomo.secret.yaml -p MATOMO_API_KEY=<Matomo access token> | oc apply -f -
+
+oc process -n devhub-tools -f eventbrite.secret.yaml -p EVENT_BRITE_API_KEY=<Event brite api key if you have> |oc apply -f –
+```
+We are currently not using meetup, but meetup token has to be there, so create one with any API token:
```

Comment:
The reason we are not using meetup is because they have _disabled_ access tokens. It is currently not possible to create this secret right now. They may change this at some point though.

I'd add a quote explaining this 

> At this time it is __NOT__ possible to get a Meetups token. They have disabled access tokens and so this step can be skipped